src/utils/openExternal.ts

```diff
@@ -3,14 +3,8 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { commands, Uri } from 'vscode';
-import { localize } from '../localize';
+import * as vscode from 'vscode';
 
 export async function openExternal(path: string, throwErrorOnFailure: boolean = false): Promise<void> {
```

Comment:
We probably should get rid of this wrapper altogether. Also, `throwErrorOnFailure` parameter is now not used at all.